views/settings.hbs

```diff
@@ -0,0 +1,71 @@
+<div class="alert alert-secondary mb-3" role="alert">
+  Please copy the values from your XcooBee account or from your downloaded key files.
+</div>
+<form action="#" class="mt-3 settings-form">
+  <div class="row m-0">
+    <div class="col-sm-4 p-2">
+      <h6>Needed for Polling</h6>
+      <a href="https://testapp.xcoobee.net/user/settings/security">Open XcooBee API Settings</a>
```

Comment:
I am not sure what the correct answer is but do we want to use the test system or the real production system?

Note: If we change to use the real prod system, there are 2 other references below.

In the README.md file there is a Prerequisites section that states you will need a XcooBee account. If it should be an account on the test system, then let's update the documentation to clarify that.